scaletest/dashboard/chromedp.go

```diff
@@ -0,0 +1,181 @@
+package dashboard
+
+import (
+	"context"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/chromedp/cdproto/cdp"
+	"github.com/chromedp/cdproto/network"
+	"github.com/chromedp/chromedp"
+	"golang.org/x/xerrors"
+)
+
+// Action is just a function that does something.
+type Action func(ctx context.Context) error
+
+// Selector locates an element on a page.
+type Selector string
+
+// Label identifies an action.
+type Label string
+
+// defaultSelectors is a map of labels to selectors.
+var defaultSelectors = map[Label]Selector{
+	"workspaces_list":            `nav a[href="/workspaces"]:not(.active)`,
+	"templates_list":             `nav a[href="/templates"]:not(.active)`,
+	"users_list":                 `nav a[href^="/users"]:not(.active)`,
+	"deployment_status":          `nav a[href="/deployment/general"]:not(.active)`,
+	"starter_templates":          `a[href="/starter-templates"]`,
+	"workspaces_table_row":       `tr[role="button"][data-testid^="workspace-"]`,
+	"workspace_builds_table_row": `tr[role="button"][data-testid^="build-"]`,
+	"templates_table_row":        `tr[role="button"][data-testid^="template-"]`,
+	"template_docs":              `a[href^="/templates/"][href$="/docs"]:not([aria-current])`,
+	"template_files":             `a[href^="/templates/"][href$="/files"]:not([aria-current])`,
+	"template_versions":          `a[href^="/templates/"][href$="/versions"]:not([aria-current])`,
+	"template_embed":             `a[href^="/templates/"][href$="/embed"]:not([aria-current])`,
+	"template_insights":          `a[href^="/templates/"][href$="/insights"]:not([aria-current])`,
+}
+
+// ClickRandomElement returns an action that will click an element from the given selectors at random.
+// If no elements are found, an error is returned.
+// If more than one element is found, one is chosen at random.
+// The label of the clicked element is returned.
+func ClickRandomElement(ctx context.Context) (Label, Action, error) {
```

Comment:
Could you describe the use case? Basically, I'm checking if we should go with a "random click" or a repeatable/regular one would be a better fit.